### PR TITLE
PR policy

### PR DESCRIPTION
**Pull request by tomjaguarpaw**
I think it would be helpful to have the two-reviewers policy (as [explained by @TikhonJelvis](https://github.com/haskell-infra/www.haskell.org/pull/48#issuecomment-743793498)) clearly stated. This PR builds on https://github.com/haskell-infra/www.haskell.org/pull/50. If it didn't it would lead to a merge conflict.

### CODE REVIEW

**Comment by TikhonJelvis** [comment]
Hey, this is a great suggestion. I also realized that we should codify that policy as a [committee proposal](https://github.com/haskell-org/committee/pull/3) as part of our efforts to be more transparent. Once that proposal is accepted, I'll update this PR with a link to it so people can read the details.

**Review by emilypi — APPROVED** [review]

**Comment by ketzacoatl** [comment]
@TikhonJelvis does this need another approval before it can be merged, or what's blocking it?

**Comment by tomjaguarpaw** [comment]
Looks like this proposal has not been brought before the committee https://github.com/haskell-org/committee/blob/aa8db0b576f013ec129c1e50dfd7a8076a7cec21/proposals/0003-pr-process.md

**Comment by ketzacoatl** [comment]
What are next steps?

**Comment by tomjaguarpaw** [comment]
We have a committee meeting next week.  I will raise it then.

**Comment by tomjaguarpaw** [comment]
I'm not sure this is necessary now that we have https://github.com/haskell-org/committee/blob/main/proposals/0003-pr-process.md. Perhaps this PR should just contain a link to that page.

**Comment by rebeccaskinner** [comment]
I agree, I think linking to the other pr process would be good here.

**Comment by tomjaguarpaw** [comment]
> I agree, I think linking to the other pr process would be good here.

OK done. @rebeccaskinner @TikhonJelvis Would you mind looking at this one?

**Review by TikhonJelvis — APPROVED** [review]

**Review by rebeccaskinner — APPROVED** [review]

**Pull request merged**
